kura/org.eclipse.kura.linux.position/src/main/java/org/eclipse/kura/linux/position/PositionServiceImpl.java

```diff
@@ -194,16 +199,34 @@ public String getNmeaDate() {
 
     @Override
     public LocalDateTime getDateTime() {
-        if (this.currentProvider != null) {
-            return this.currentProvider.getDateTime();
+
+        if (this.options.isEnabled()) {
+
+            if (!this.options.isStatic()) {
```

Comment:
What happens if the currentProvider is null? Should we return a null as in the previous implementation?